partitura/io/importmusicxml.py

```diff
@@ -456,13 +456,14 @@ def _parse_parts(document, part_dict):
         #     shift.applied = True
 
 
-def _handle_measure(measure_el, position, part, ongoing, doc_order):
+def _handle_measure(measure_el, position, part, ongoing, doc_order, measure_counter):
+    # print('_handle_measure') # NOTE del
     """
```

Comment:
Maybe it would be worthwhile to document the function a little bit, since there is a new parameter (`measure_counter`). It would be nice to fully document all functions at some point 😉